packages/frontend/themes/_themebase.scss

```diff
@@ -184,6 +184,7 @@ $hover-contrast-change: 2%;
   --globalLinkColor: #2090ea; // Manual value: This will not get generated in the future
   --globalBackground: #{$bgPrimary};
   --globalText: #{$textPrimary};
+  --textLessContrast: #{$colorLessContrast}; // less contrast to background than global text
```

Comment:
Introducing a new generic text color with a meaningful name should avoid to have too many similar colors. So anytime you want to have less contrast for a text you can use this variable. We could also rename it to hintTextColor to be more specific (?)
TBD: this is in contrast to the suggestion in https://github.com/deltachat/deltachat-desktop/issues/3018
Which says all css variable should be per component only.
That would end up in having too many variables imho